clis/teliod/src/qnap.rs

```diff
@@ -0,0 +1,239 @@
+use std::{
+    env, fs,
+    io::{self, Read},
+    process::{Command, Stdio},
+};
+
+use serde_json::json;
+
+use crate::config::TeliodDaemonConfig;
+
+struct PathConfig;
+
+impl PathConfig {
+    const APP_TMP_DIR: &'static str = "/tmp/nordsecuritymeshnet";
```

Comment:
NIT: Would be nice to have them configurable via parameters, but leave default's as is.